charts/vastcsi/values.yaml

```diff
@@ -63,6 +63,8 @@ storageClassDefaults:
   mountOptions: []
   # Name of QoS policy associates with the view.
   qosPolicy: ""
+  # Reclaim policy to use with the storage class.
+  reclaimPolicy: ""
```

Comment:
@aveuiller - we're thinking to make this `Delete`, since that would be the default behavior provided by the storage-class when this attribute is missing. what do you think?

```suggestion
  reclaimPolicy: "Delete"
```